docs/guides/new_players/getting_started.mdx

```diff
@@ -0,0 +1,134 @@
+
+<details>
+<summary>Spoilers on how to solve public scripts.</summary>
+
+To solve public scripts, take ((weyland.public)) for example. When a public script is called without any arguments, it will display a default welcome message. Below is that welcome message for ((weyland.public)).
+
+```
+888       888 8888888888 Y88b   d88P 888             d8888 888b    888 8888888b.  
+888   o   888 888         Y88b d88P  888            d88888 8888b   888 888  "Y88b 
+888  d8b  888 888          Y88o88P   888           d88P888 88888b  888 888    888 
+888 d888b 888 8888888       Y888P    888          d88P 888 888Y88b 888 888    888 
+888d88888b888 888            888     888         d88P  888 888 Y88b888 888    888 
+88888P Y88888 888            888     888        d88P   888 888  Y88888 888    888 
+8888P   Y8888 888            888     888       d8888888888 888   Y8888 888  .d88P 
+888P     Y888 8888888888     888     88888888 d88P     888 888    Y888 8888888P"  
+
+
+
+Welcome to Weyland Yutanihama's public information script. Please refrain from engaging in criminal activity.
+latest | what_we_do | 
+```
+
+Going off of what trust has said in the vLAN about trying to call scripts with empty arguments (like so: {}), if we do that with ((weyland.public)), we get the following output:
+
+```
+>>weyland.public {}
+Please specify a command with navigation:"<command name>"
+Public commands are "what_we_do" "latest" 
+-- access directory with navigation:"list"
+```
+
+Depending on when the public is ran, the output may be different due to the fact that the public script may have shifted (which means it's output has changed) from the time of writing this. Now we know that the way we input commands is using the "navigation" key. If we then try to access one of the three available commands using the "navigation" key, for example "what_we_do", we get the following output:
+
+```
+>>weyland.public {navigation:"what_we_do"}
+In the mood to put something a little different in your stomach?  Come out to Weyland Yutanihama! The freshest raw food in to
+wn.
+We are calling this strategy bethebest and we will continue to strive to deliver on this promise.
+```
+
+Take note of the "strategy". Next, run one of the two other available commands, "list".
+
+```
+>>weyland.public {navigation:"list"}
+No password specified.
+```
+Here, it states that a password is required. This is where that "strategy" is used. The keys that can be used for specifying the password are: "password", "pass" and "p." If we try and use one of those to specify the password, which in this case is the strategy from above, "bethebest", we get this:
+
+```
+>>weyland.public {navigation:"list", password:"bethebest"}
+Authenticated. Please specify a project to get a member list.
+```
+The aforementioned member list is where a player would obtain NPC locs. To access it, we need to specify a project. For that, run the final command that hasn't been executed yet, "latest."
```

Comment:
I would place the period outside the quotes to not confuse people into thinking it's part of the command:
```suggestion
The aforementioned member list is where a player would obtain NPC locs. To access it, we need to specify a project. For that, run the final command that hasn't been executed yet, "latest".
```